numpy/linalg/lapack_lite/clapack_scrub.py

```diff
@@ -305,7 +305,7 @@ def scrubSource(source, nsteps=None, verbose=False):
     else:
         nsteps = None
 
-    source = scrub_source(source, nsteps, verbose=True)
+    source = scrubSource(source, nsteps, verbose=True)
```

Comment:
Huh. How did this work?

Reply:
I guess this Python 2.7 script hasn't been used for a long time - probably not after f589305da2758f28eacc33fe0e50f1c1460a4c5e in 2018.